docs/intro/economics.md

```diff
@@ -43,7 +43,7 @@ issuance over time. Next is a brief section on
 [Adjusted Staking Yield](/docs/economics/inflation/_adjusted_staking_yield.md),
 and how token dilution might influence staking behavior.
 
-An overview of [Transaction Fees](/docs/core/transactions/fees.md) on Solana is
+An overview of [Transaction Fees](/docs/intro/transaction_fees) on Solana is
```

Comment:
this link is invalid. it should be `/docs/intro/transaction-fees`

Reply:
change to `/docs/core/fees` for the updated fee page